src/NuGet.Core/NuGet.PackageManagement/GlobalSuppressions.cs

```diff
@@ -176,3 +176,4 @@
 [assembly: SuppressMessage("Build", "CA2237:Add [Serializable] to PackageReferenceRollbackException as this type implements ISerializable", Justification = "<Pending>", Scope = "type", Target = "~T:NuGet.PackageManagement.PackageReferenceRollbackException")]
 [assembly: SuppressMessage("Build", "CA1067:Type NuGet.ProjectManagement.FileTransformExtensions should override Equals because it implements IEquatable<T>", Justification = "<Pending>", Scope = "type", Target = "~T:NuGet.ProjectManagement.FileTransformExtensions")]
 [assembly: SuppressMessage("Usage", "CA2208:Instantiate argument exceptions correctly", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.PackageManagement.NuGetPackageManager.PreviewProjectsUninstallPackageAsync(System.Collections.Generic.IReadOnlyCollection{NuGet.ProjectManagement.NuGetProject},System.String,NuGet.PackageManagement.UninstallationContext,NuGet.ProjectManagement.INuGetProjectContext,System.Threading.CancellationToken)~System.Threading.Tasks.Task{System.Collections.Generic.IEnumerable{NuGet.PackageManagement.NuGetProjectAction}}")]
+[assembly: SuppressMessage("Usage", "CA2208:Instantiate argument exceptions correctly", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.PackageManagement.NuGetPackageManager.PreviewBuildIntegratedProjectsActionsAsync(System.Collections.Generic.IReadOnlyCollection{NuGet.ProjectManagement.Projects.BuildIntegratedNuGetProject},System.Collections.Generic.Dictionary{System.String,NuGet.PackageManagement.NuGetProjectAction[]},NuGet.Packaging.Core.PackageIdentity,System.Collections.Generic.IReadOnlyCollection{NuGet.Protocol.Core.Types.SourceRepository},NuGet.ProjectManagement.INuGetProjectContext,System.Threading.CancellationToken)~System.Threading.Tasks.Task{System.Collections.Generic.IEnumerable{NuGet.PackageManagement.ResolvedAction}}")]
```

Comment:
This will be removed once https://github.com/nuget/client.engineering/issues/642 is solved

Reply:
FYI, I have an active PR that will remove all CA2208 https://github.com/NuGet/NuGet.Client/pull/3799